commercio-3/.data.json

```diff
@@ -0,0 +1,8 @@
+{
+  "Name": "3",
+  "Release": "v5.0.0",
+  "Seeds": "8d3ac30a774245019f7b94d7a0713fd8caccaa03@seed-01.commercio.network:26656,68ccfc9a1574923e344993d49253d8169f592f9e@seed-02.commercio.network:26656",
+  "Persistent_peers": "3f26a7d3269e352574e760d4b08d045aa40ebbf2@persistent-01.commercio.network:26656,49fdf20f845573429a695c15729c60f52bb5ab19@84.46.241.230:26656,542a55fff599ea07e2b4841febff4a78cf8db2aa@persistent-05.commercio.network:26656,01f615a56a7ba14d39e4eaf2bfd00a885cc4e629@persistent-03.commercio.network:26656,15767f4961b501eb67e740aa60c1608bf65679f3@persistent-02.commercio.network:26656,d6e07b5f0a69bd4d6f4e6e931f00fbcb992a8654@persistent-04.commercio.network:26656,e2bd5ac83f0d2fde72571568b9a4203f7e76067d@64.225.95.231:26656",
+  "Genesis_Checksum": "a4a8a70f471aaef36f820996a0bcbb6b74f7ac1ff0756d0de63ec1db21e4f2f8",
+  "Ordered_Checksum": "de4306aa45f491ac2dc29ba82f6e18db6dd6f58032a67801bcfa88dd6668b25e"
+}
```

Comment:
Maybe it is better to add rpcs server here, so you can use it in the chain installer tool. Something like this
```
...
   "rpc_servers": "rpc-mainnet.commercio.network:80,rpc2-mainnet.commercio.network:80"
}
```

or 

```
...
   "rpc_servers": [
       "rpc-mainnet.commercio.network:80",
       "rpc2-mainnet.commercio.network:80"
   ]
}
```